### DISCUSSION TITLE
Airtable Create query failing

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#7280] [Help] [answered]
Hello - I've seem to be having some trouble with setting up a **Create** query on the airtable integration. 

In this first example, I can successfully create a new record in my airtable if I enter the record variables directly in the query body. 
<img width="1570" alt="Screenshot 2022-08-15 at 17 31 40" src="https://user-images.githubusercontent.com/64409787/184676251-e30de840-ef72-40ba-bf4a-8abc515e2ff9.png">

In this second example, I've used bindings in place of the 'hard-coded' variables and set default values for all. If I try to run the query I get the error shown in the image. 
<img width="1526" alt="Screenshot 2022-08-15 at 17 31 20" src="https://user-images.githubusercontent.com/64409787/184676708-0e0e1371-2f60-49b9-a066-432af4a72788.png">

Any ideas? I don't think I've missed anything out of the query in the second example. 

Thanks

**Comment by melohagan** [comment]
Hi @adamjdavies 

Your bindings look a little strange to me. For example this is not a valid binding: `{{[comment date]}}`
It should be `{{comment date}}` without the square brackets.
Same goes for Trade-in order.

**Comment by ghost** [comment]
Hi @melohagan  - thanks for getting back to me on this. 

Still getting the same error even if I update the bindings. Again, it works if I enter values directly into the query, but not when bindings & default values are being used for some reason. 

<img width="1570" alt="Screenshot 2022-08-18 at 09 27 27" src="https://user-images.githubusercontent.com/64409787/185348100-6109a29a-811d-4790-b82a-ab431c41b97b.png">

<img width="1570" alt="Screenshot 2022-08-18 at 09 28 35" src="https://user-images.githubusercontent.com/64409787/185348388-4d5e94ab-98cd-4be1-b980-26459bb3bf5a.png">


Thanks

**Reply by melohagan** [reply]
Great, cheers for that. I'll look into this further!

**Reply by ghost** [reply]
Perfect, thanks!

I'm also struggling to understand the default number of records returned in a Read query for Airtable (more info in this ticket #7312). Could you provide some clarity on this?

**Comment by melohagan** [comment, marked as answer]
@adamjdavies This bug is actually specific to the use of the binding `{{ user }}`
This is a reserved binding. If you rename that to something like `username` it should work fine. 
I've picked up the bug you raised: https://github.com/Budibase/budibase/issues/7322

**Reply by ghost** [reply]
Perfect. Thanks for looking to this. All working now if I use an updated 'user' binding name!

**Reply by melohagan** [reply]
You're welcome - glad you're sorted!
